docs/docsource/Reference/bucket_lifecycle_operations/put_bucket_lifecycle.rst

```diff
@@ -424,25 +422,24 @@ configuration:
    |                       |                       |                       |
    |                       | **Ancestor:** Rule    |                       |
    |                       |                       |                       |
-   |                       | **Valid values:**     |                       |
+   |                       | **Valid Values:**     |                       |
    |                       | Enabled or Disabled.  |                       |
    +-----------------------+-----------------------+-----------------------+
    | StorageClass          | Specifies the storage | Yes                   |
-   |                       | class to which you    |                       |
-   |                       | want the object to    | This element is       |
-   |                       | transition.           | required only if you  |
-   |                       |                       | specify one or both   |
-   |                       | **Type:** String      | its ancestors.        |
+   |                       | class (Zenko  	   | 			   |
+   |			   | location) to which	   | This element is       |
```

Comment:
Same potential misalignment here

Reply:
FWIW, if that were in the code, tox would gag on it.